WalletWasabi/Models/ChaumianCoinJoin/CcjRound.cs

```diff
@@ -720,8 +720,21 @@ private async Task TryOptimizeFeesAsync(IEnumerable<Coin> spentCoins)
 				int estimatedSigSizeBytes = UnsignedCoinJoin.Inputs.Count * Constants.P2wpkhInputSizeInBytes;
 				int estimatedFinalTxSize = UnsignedCoinJoin.GetSerializedSize() + estimatedSigSizeBytes;
 				Money fee = UnsignedCoinJoin.GetFee(spentCoins.ToArray());
+
 				// There is a currentFeeRate null check later.
-				FeeRate currentFeeRate = fee is null ? null : new FeeRate(fee, estimatedFinalTxSize);
+				FeeRate currentFeeRate = null;
+				if (fee is null)
```

Comment:
Concept ACK. Format idea: do you like switch/case statements?

Reply:
I don't think a `switch` statement can make any difference here.

Reply:
I prefer switch if there are more then 2 if else branches.

Reply:
I agree it should be a switch, because that'd look better. But I don't agree with @molnard's reasoning.